<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>focusercontrollerwidget</class>
 <widget class="QWidget" name="focusercontrollerwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>115</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>115</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>115</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QComboBox" name="focuserSelectionBox"/>
   </item>
   <item>
    <widget class="QWidget" name="focuserInfo" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="targetLabel">
        <property name="text">
         <string>Target position:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="currentLabel">
        <property name="text">
         <string>Current position:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="currentField">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="positionSpinBox">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
        <property name="value">
         <number>32768</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="2">
       <widget class="QPushButton" name="positionButton">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1000</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>Move</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>positionButton</sender>
   <signal>clicked()</signal>
   <receiver>focusercontrollerwidget</receiver>
   <slot>guiChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>positionSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>focusercontrollerwidget</receiver>
   <slot>guiChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>focuserSelectionBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>focusercontrollerwidget</receiver>
   <slot>focuserChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>positionSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>focusercontrollerwidget</receiver>
   <slot>editingFinished()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>guiChanged()</slot>
  <slot>focuserChanged(int)</slot>
  <slot>editingFinished()</slot>
 </slots>
</ui>
